tests/test_tuya.py

```diff
@@ -462,25 +462,25 @@ async def test_zonnsmart_state_report(zigpy_device_from_quirk, quirk):
         tuya_cluster.handle_message(hdr, args)
 
     assert len(thermostat_listener.cluster_commands) == 0
-    assert len(thermostat_listener.attribute_updates) == 11
+    assert len(thermostat_listener.attribute_updates) == 9
```

Comment:
So two attributes no longer update when the TRV reports its state? Or what is the reason this number changes?

Reply:
Good catch, I found the issue and fixed the code instead :)